### PR TITLE
CORE-3560: created a hover tooltip to display each link's titles

### PR DESCRIPTION
**Pull request by itslong**
Created a css/components.css file to track component-specific (tooltip, in this ticket) css. Will refactor to individual files if the project increases in size.

### CODE REVIEW

**Comment by itslong** [comment]
Ignore the changes in the script.js as work was paused midway to complete the sass file for tooltip.

**Comment by itslong** [comment]
had compass compile errors related to "import chroma/functions not found" even though the config.rb file was pointed to the node_modules path (where chroma was installed). 

fix: npm install in the subtheme directory since the node_modules path is NOT being checked in. Then set compass watch one layer ABOVE the subtheme so "compass watch movie" can watch for new sass.

File: drupal/web/sites/all/themes/movie/css/components/components.css [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.tooltip {
```
Comment (longlivelance) [inline drupal/web/sites/all/themes/movie/css/components/components.css]: We can delete this file.

File: drupal/web/sites/all/themes/movie/sass/components/footer/_footer.scss [hunk 2]
```diff
@@ -1,14 +1,10 @@
...
-.footer,
-%footer {
-  @extend %clearfix;
-  // Wrapper for any blocks placed in the header region.
-  &__region {
-    // Clear the logo.
-    clear: both;
-  }
-}
+.tooltip {
```
Comment (longlivelance) [inline drupal/web/sites/all/themes/movie/sass/components/footer/_footer.scss]: Why are we duplicating this scss in the _footer file?

File: drupal/web/sites/all/themes/movie/sass/components/tooltip/tooltip.scss [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.tooltip {
```
Comment (longlivelance) [inline drupal/web/sites/all/themes/movie/sass/components/tooltip/tooltip.scss]: I think this file should actually be `_tooltip.scss`.

File: drupal/web/sites/all/themes/movie/js/script.js [hunk 4]
```diff
@@ -17,10 +17,26 @@
   // To understand behaviors, see https://drupal.org/node/756722#behaviors
   Drupal.behaviors.my_custom_behavior = {
     attach: function (context, settings) {
+      // item-list
+      $('.item-list ul li a').hover(function () {
+        var title = $(this).text();
 
-      // Place your code here.
+        $('<p class="tooltip"></p>')
+        .text(title)
```
Comment (longlivelance) [inline drupal/web/sites/all/themes/movie/js/script.js]: When chaining functions, you should follow tab rules.  This code block should then become:

```
$('<p class="tooltip"></p>')
  .text(title)
  .appendTo('body')
  .fadeIn('fast');
```

File: drupal/web/sites/all/themes/movie/sass/components/footer/_footer.scss (hunk 2 above)
Comment (itslong) [reply drupal/web/sites/all/themes/movie/sass/components/footer/_footer.scss]: I had copied the tooltip into the wrong file. Thanks for pointing this out

File: drupal/web/sites/all/themes/movie/js/script.js [hunk 5]
```diff
@@ -17,10 +17,26 @@
...
+        .appendTo('body')
+        .fadeIn('fast');
+      }, function() {
+        $('.tooltip').remove();
+      }).mousemove(function (e) {
+        // Get X, Y coordinates of mouse
+        var mousex = e.pageX + 20;
+        var mousey = e.pageY + 10;
+        $('.tooltip')
```
Comment (longlivelance) [inline drupal/web/sites/all/themes/movie/js/script.js]: Same as above with this one. Should become: 

```
 $('.tooltip')
  .css({ top: mousey, left: mousex });
```

**Pull request merged**